website/docs/d/nsxt_app_port_profile.html.markdown

```diff
@@ -53,7 +58,10 @@ The following arguments are supported:
 
 * `org` - (Optional) The name of organization to use, optional if defined at provider level. Useful
   when connected as sysadmin working across different organisations.
-* `vdc` - (Optional) The name of VDC to use, optional if defined at provider level.
+* `vdc` - (Deprecated; Optional) The name of VDC to use, optional if defined at provider level.
+  Deprecated and replaced by `context_id`
+* `context_id` - (Optional) ID of NSX-T Manager, VDC or VDC Group. Replaces deprecated fields `vdc`
+  and accepts VDC, VDC Group or NSX-T Manager ID. 
```

Comment:
Looks like we were duplicating statements and not sure why plural "fields" was used. What about this:

```suggestion
* `context_id` - (Optional) ID of NSX-T Manager, VDC or VDC Group. Replaces deprecated field `vdc`.
```